src/development/platform-integration/ios/apple-frameworks.md

```diff
@@ -45,139 +47,29 @@ Occasionally, you might also need to change app settings or
 initialization logic- which should be explained in the package's 
 "Readme" page on [pub.dev][].
 
-### Accessing the photo library
-In Swift and Objective-C, you use `PhotoKit` and the `PhotosUI` framework
-to access user photos.
+### Flutter Plugins and Apple Frameworks
+
+| Use Case | Apple Framework or Class | Flutter Plugin |
+| -------- | --------------- | -------------- |
+| Accessing the photo library | `PhotoKit` (`Photos` and `PhotosUI ` frameworks) | [`image_picker`][] |
+| Accessing the camera | `UIImagePickerController` class with the `.camera` `sourceType` | [`image_picker`][] |
```

Comment:
`UIImagePickerController` can also be used for accessing photo library. maybe also add this to previous row?

Reply:
maybe also add a row 
```
| More advanced camera features | AVFoundation | camera plugin |
```

Reply:
Added, thanks!